_pages/about-us/tts-history.md

```diff
@@ -12,6 +12,19 @@ While OPP was paving the way for government shared services and open data, there
 
 That project became [18F](https://18f.gsa.gov/) when the office officially launched in March 2014. 18F quickly grew in size to help federal agencies work on technology projects while learning new skills. In 2016, 18F, PIF, and OPP came together under the newly-formed Technology Transformation Service office. In 2017, TTS added the [IT Modernization Centers of Excellence (CoEs)](https://coe.gsa.gov/) to focus on whole-agency technology modernization, and all of TTS moved into GSA’s existing Federal Acquisition Service.
 
+
+## TTS Today
+
+TTS is part of GSA's Federal Acquisition Service (FAS) and contains several offices:  
+
+- **18F** partners with agencies to provide consulting and software development services, with a focus on cultural transformation and empowerment. A critical part of 18F’s mission is to teach agencies new digital skills during projects so they can take over continued development.
```

Comment:
Would it be worth mentioning cloud.gov and login.gov?

Reply:
in the interest of aligning with the join.tts.gov language, and because they are not (at least as far as I know) separate offices, I think we shouldn't, but should be sure they're accounted for elsewhere. I'm pretty sure they are :-)